exasol/slc_ci/lib/get_build_config_model.py

```diff
@@ -0,0 +1,8 @@
+from pathlib import Path
+
+from exasol.slc_ci.model.build_config_model import BuildConfig
+
+
+def get_build_config_model() -> BuildConfig:
+    build_config_path = Path.cwd() / "build_config.json"
```

Comment:
Instead of using cwd, we probably need to do a backwards search, if we want to make it properly, however, the question is this needed here, because the project is mainly used in a well defined environment.

Reply:
Added a backwards search